Macros.h

```diff
@@ -126,7 +126,8 @@ static inline bool isNaN(double x) {
    return !isgreaterequal(x, x);
 }
 
-/* Checks if x is nonnegative. Returns false if x is NaN. */
+/* Checks if x is nonnegative (with -0.0 considered nonnegative).
+   Returns false if x is NaN. */
```

Comment:
Possible wording?

```suggestion
/* Checks if x is nonnegative by checking x >= 0.0.
   Due to IEEE754 having a duplicate zero -0.0 is considered as nonnegative.
   Returns false if x is NaN. */
```

Reply:
I've shortened your suggestion a bit. Let's see if the new description looks better.